src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -1202,6 +1203,131 @@ describe('Form Model', () => {
         })
       })
     })
+
+    describe('updateFormLogic', () => {
+      const logicId1 = new ObjectId().toHexString()
+      const logicId2 = new ObjectId().toHexString()
+
+      const mockExistingFormLogic = {
+        form_logics: [
+          {
+            _id: logicId1,
+            id: logicId1,
```

Comment:
I'm in favour of standardising to `_id` because it would remove the need for stuff like this

Reply:
edited